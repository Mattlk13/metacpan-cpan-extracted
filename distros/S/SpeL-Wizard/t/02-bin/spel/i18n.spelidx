format|ogg
audiodir|spel
chunkdir|spel
language|english
macad|numpower|2|-NoValue-|preceding ##1 @{i18n(Power,##2)} trailing
chunk|i18n
