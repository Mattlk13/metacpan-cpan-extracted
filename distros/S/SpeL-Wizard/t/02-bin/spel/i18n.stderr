spel-wizard.pl - v0.9 - Walter Daems <walter.daems@uantwerpen.be>
=================================================================
- Reading configuration file from /home/wdaems/perl5/lib/perl5/auto/share/dist/SpeL-Wizard/tts.conf
- no t/02-bin/spel/i18n.aux available at /home/wdaems/lab/spelbox/perl/lib/SpeL/Wizard.pm line 96.
- Treating t/02-bin/spel/spel/i18n.tex                             
  Parsing t/02-bin/spel/spel/i18n.tex                       [  0%]  Parsed  t/02-bin/spel/spel/i18n.tex                       [  0%]$Parsetree = bless( {
  'Element' => [
    bless( {
      '' => 'This is a simple i18n test: '
    }, 'SpeL::Object::TokenSequence' ),
    bless( {
      'Args' => [
        bless( {
          'Element' => [
            bless( {
              '' => '2'
            }, 'SpeL::Object::TokenSequence' )
          ]
        }, 'SpeL::Object::ElementList' ),
        bless( {
          'Element' => [
            bless( {
              '' => '7'
            }, 'SpeL::Object::TokenSequence' )
          ]
        }, 'SpeL::Object::ElementList' )
      ],
      'Name' => 'numpower',
      'trailingws' => '


'
    }, 'SpeL::Object::Command' )
  ]
}, 'SpeL::Object::ElementList' );

Done.
