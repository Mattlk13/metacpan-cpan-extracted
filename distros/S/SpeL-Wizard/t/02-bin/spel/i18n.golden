This is a simple i18n test: preceding 2 to the power of 7 trailing
